client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -573,6 +567,40 @@ export class CheckoutThankYou extends Component<
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
+		/** REFACTORED REDESIGN **/
+
+		if ( isRefactored( this.props ) ) {
+			let pageContent = null;
+
+			if ( ! wasBulkDomainTransfer && isDomainOnly( purchases ) ) {
+				pageContent = <DomainOnlyThankYou purchases={ purchases } />;
+			}
+
+			if ( pageContent ) {
+				const siteId = this.props.selectedSite?.ID;
+				const siteSlug = this.props.selectedSite?.slug;
+
+				return (
+					<Main className="is-redesign-v2">
+						<PageViewTracker { ...this.getAnalyticsProperties() } title="Checkout Thank You" />
+
+						{ this.isDataLoaded() && siteId && <QuerySitePurchases siteId={ siteId } /> }
+
+						<MasterbarStyled
+							onClick={ () => page( `/home/${ siteSlug ?? '' }` ) }
+							backText={ translate( 'Back to home' ) }
+							canGoBack={ !! siteId }
+							showContact={ true }
```

Comment:
`={ true }` is redundant

```suggestion
							showContact
```